projects/ppwcode/ng-common-components/src/lib/table/column-directives/ppw-column.directive.ts

```diff
@@ -36,6 +36,12 @@ export class PpwColumnDirective<TRecord> {
     /** The label of the column, this is shown in the header of the column if no header template is set. */
     public label: InputSignal<string | undefined> = input<string | undefined>(undefined)
 
+    /** Whether the column should be sticky. */
+    public sticky: InputSignal<boolean> = input<boolean>(false)
```

Comment:
Add the boolean attribute transform function from Angular to both input bindings, this will allow us to do `<ppw-column sticky ...>` instead of `<ppw-column [sticky]="true" ...>`.